Library/UIButton+HapticFeedback.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+extension UIButton {
+
+  @available(iOS 10.0, *)
+  public func generateSelectionFeedback() {
+    UISelectionFeedbackGenerator().selectionChanged()
+  }
+
+  @available(iOS 10.0, *)
+  public func generateSuccessFeedback() {
+    UINotificationFeedbackGenerator().notificationOccurred(.success)
```

Comment:
This could be replaced with `UIFeedbackGenerator().ksr_successFeedback()` in order to not repeat code